docs/source/design/warehousing.rst

```diff
@@ -67,7 +67,8 @@ one of that stage's returned DataFrames.
 
 .. image:: /_static/images/warehouse_inside_db.png
 
-.. note::
+Lifecycle
+=========
 
-    Details for the warehouse used for ingesting Kids First studies can be
```

Comment:
Can we leave the link to the warehouse readme? It has the connection details that are specific to KF. Maybe it also redundantly mentions how data is cleared out but I think its ok to be redundant.

Reply:
the https://github.com/d3b-center/clinical-data-flow/tree/master/warehouse#data-organization section there has wrong information

Reply:
Yea can you make a PR for it to correct it?